backend/src/test/java/com/pickpick/service/ChannelSubscriptionServiceTest.java

```diff
@@ -11,16 +16,16 @@
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.util.Comparator;
 import java.util.List;
-import java.util.stream.Collectors;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 @SpringBootTest
 @Transactional
 class ChannelSubscriptionServiceTest {
 
+    private static final long NOT_EXISTED_CHANNEL_ID = 1L;
     @Autowired
```

Comment:
```suggestion
    private static final long NOT_EXISTED_CHANNEL_ID = 1L;
    
    @Autowired
```
메서드랑 상수 사이에 줄바꿈 하나 있으면 좋을 것 같아요~~😄👍